docs/Frequently-Asked-Question.md

```diff
@@ -236,3 +236,34 @@ There are several methods to run `shiori` on start up, however the most recommen
     ```sh
     systemctl enable --now shiori
     ```
+
```

Comment:
I would add it as a separate section here:

```
### How to run `shiori` on start up (macOS)?
```

And rename the other one to specify Linux

Reply:
Made the requested changes. Please let me know if anything looks not ok. (I'm not very used to PR). Thanks.